web/modules/custom/contact_information/contact_information.module

```diff
@@ -51,3 +49,42 @@ function contact_information_inline_entity_form_table_fields_alter(&$fields, $co
     $fields['label']['label'] = t('Name');
   }
 }
+
+/**
+ * Implements hook_inline_entity_form_entity_form_alter().
+ */
+function contact_information_inline_entity_form_entity_form_alter(&$entity_form, &$form_state) {
+  // Add an entity builder callback to fill the label field with the values of
```

Comment:
Again, this should go in the function implementing it and use a simpler "to fill the label" or similar.